src/app/item.service.ts

```diff
@@ -32,14 +33,14 @@ export class ItemService {
 
     const items = of(Mock)
 //    AutoId.newId();
-//    return this.items$;
-  	return items;
+    return this.items$;
+//  	return items;
   }
 
   getItem(itemID: string): Observable<item>{
```

Comment:
In `ItemService.getItem()` I would recommend doing a single database query for the specific item you are looking for rather than searching through the items locally. This is because there are cases where you might not have any values in the `items$` observable. Also this will work better if you have a route for a specific item (eg. `/items/:id`) since you won't be depending on `items$` emitting a value.